.github/workflows/deploy_sdist.yaml

```diff
@@ -0,0 +1,27 @@
+name: Deploy sdist
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v5
+
+      - name: Create sdist
+        shell: bash -l {0}
+        run: |
+          python -m pip install -q build
+          python -m build -s
```

Comment:
I had a coworker point this out, you should remove the `-s` so a wheel is also built. This makes installation faster for pure-python packages since `pip install` doesn't have to do it at install time.